### ISSUE TITLE
Array<Float32> resolves to List<float> in java causing "Type argument cannot be of primitive type" error

### ISSUE THREAD

**Issue by liangjingoforit** [ardielle/ardielle-tools#32]
ex:
rdl definistion:
```
type values Array<Float32> 
```
can only resolve to  
```
public List<float> values 
```
 instead of   
```
public List<Float> values
```
which will have error of  "Type argument cannot be of primitive type" in java
Please help resolve this issue.

Thanks,
Liang

**Comment by liangjingoforit** [comment]
Any updates for this issue?

**Comment by boynton** [comment]
Fixed in change #33
